modules/images-managing-images-enabling-imagestreams-kube.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/managing-images/using-imagestreams-with-kube-resources.adoc
+
+
+[id="images-managing-images-enabling-imagestreams-kube_{context}"]
+= Enabling imagestreams with Kubernetes resources
+
+When using this feature, you can only reference imagestreams that reside in the
+same project as the resource. The imagestream reference must consist of a single
+segment value, for example `ruby:2.5`, where `ruby` is the name of an
+imagestream that has a tag named `2.5` and resides in the same project as the
+resource making the reference.
+
+[NOTE]
+====
+This feature can not be used in the "default" namespace, nor in any "openshift-"
+or "kube-" namespace.
+====
+
+.Procedure
+
+There are two ways to enable this:
+
+* Enabling imagestream resolution on a specific resource. This allows only this
+  resource to use the imagestream name in the image field.
+* Enabling image stream resolution on an imagestream. This allows all
+  resources pointing to this imagestream to use it in the image field.
+
+. Both of these operations can be done using `oc set image-lookup`. For example,
```

Comment:
The numbered steps appear under the second bullet in the preview:
https://using-imagestreams-kube-resources--ocpdocs.netlify.com/openshift-enterprise/latest/openshift_images/managing-images/using-imagestreams-with-kube-resources.html#images-managing-images-enabling-imagestreams-kube_using-imagestreams-with-kube-resources